src/extract.ts

```diff
@@ -145,16 +145,28 @@ function getHumanReadableUnitValue(seconds: number): [number, string] {
     }
 }
 
-function getCommitFromPr(pr: PullRequest): Commit {
+function getCommitFromWebhookPayload(): Commit {
```

Comment:
you've renamed this function but the old name is still used in the `getCommit` function

Reply:
This was a miss in the rebase, I've fixed it now!